src/routes/user/show.rs

```diff
@@ -343,20 +419,23 @@ pub fn ShowUser() -> impl IntoView {
                                     None => {
                                         return view! {
                                             <p class="text-red-500">
-                                                "No user with the id "{user_id.0}" has been found!"
+                                                "No user with the id "
+                                                {user_id
+                                                    .get()
+                                                    .map(|id| id.0.to_string())
+                                                    .unwrap_or("None".to_string())}" has been found!"
                                             </p>
                                         }
                                             .into_any();
                                     }
                                 };
                                 let money_signal = RwSignal::new(user.money);
                                 let transactions = RwSignal::new(Vec::<Transaction>::new());
-                                let m_args = MoneyArgs {
-                                    user_id,
+                                let (user_id, _) = signal(user.id);
```

Comment:
Why is this wrapped in a Signal?

Reply:
UserArgs recieves a signal so that we can redirect from one user view to another. Also the params getter returns a signal. This is also why the resource is used below